modules.yml

```diff
@@ -1250,3 +1250,13 @@
   path: reporting/xml
   required_keys: []
   version: '1.1'
+- author: Qazeer
```

Comment:
Using the indexing script would have placed this in the proper place within the index file, which is done alphabetically by the path. Please use the script to index the module so that it is sorted properly. Thanks.